content/W2016/W16-media-stephanie.md

```diff
@@ -0,0 +1,16 @@
+Title: Stephanie Morillo Talk
+Date: 2016-03-22
+Category: Media
+Tags: talks
```

Comment:
I couldn't see any that exist that apply, but maybe woc in tech, or technical writing?